facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -183,12 +184,42 @@ trait FapiFrontPress extends Logging with ExecutionContexts {
       .map(_.map(PressedContent.make))
   }
 
+  private def collectionWithMetadata(metadata: Metadata, path: String, config: ConfigJson): Option[String] =
+    (for {
+      front <- config.fronts.get(path)
+      collectionId <- front.collections
+      collectionConfig <- config.collections.get(collectionId) if collectionConfig.metadata.exists(_.contains(metadata))
+    } yield collectionId).headOption
+
+  private def withHigPriorityCollections(parentPath: String, config: ConfigJson, collections: List[String]): List[String] =
+    collections match {
+      case head :: tail =>
+        List(
+          collectionWithMetadata(Breaking, parentPath, config),
+          Some(head),
```

Comment:
Worth noting that this means that for the old email renderer functionality to be replicated exactly, the top container must be backfilled with the canonical container on the corresponding front - which seems fine to me, though worth checking if the canonical container ever changes